app/collins/util/config/GpuConfig.scala

```diff
@@ -0,0 +1,12 @@
+package collins.util.config
+
+object GpuConfig extends Configurable {
+  override val namespace = "gpu"
+  override val referenceConfigFilename = "gpu_reference.conf"
+
+  def supportedVendorStrings= getStringSet("supportedVendorStrings", Set("NVIDIA Corporation"))
```

Comment:
TBH i feel like the reference config is a confusing layer of indirection. When trying to figure out how collins gets some default config setting, you have to unwind the magic config loading behavior (reference configs, included base configs, environment configs, then application.conf) to understand what gets what from where. I personally prefer (and recommended) to have default config directly in the code